flyingcloud/utils/docker_util.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
@amoffat instead of this code, can we use the [retrying](https://pypi.python.org/pypi/retrying) library...?

Reply:
is that like https://www.npmjs.com/package/trim ? :)  In seriousness, we're doing custom logging and catching custom exceptions.  I'm sure it is possible to do with retrying but easier to just write it, given how simple it is for our use case